### DISCUSSION TITLE
Several questions on how to customize client and scene

### DISCUSSION THREAD

**Discussion by nonew** [mozilla/hubs#3974] [Hubs Cloud Questions]
Hi, Hubs

1. if there are multiple rooms in a scene, can the client take the user to one of the rooms directly? so users don't need to look around and find that room in the scene.
2. when the user clicks an object in the scene, can the client take some customized action by programming the client code, such as show a message dialog or play a video on full screen?
3. is it possible to place a camera in the scene and when I click it, it will show a video stream sourced from my home web camera?
4. in the admin page I cannot remove imported scenes and images, the page prompts me that selection was removed, but when I come back and refresh, they are still there. Sometimes I encounter a 404 not found error while removing.

Thanks & Best regards!
Kevin

**Comment by pattersonbl2** [comment]
are you still having issues with these questions ?
